docs/_content/standaard/gegevensmodel/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Gegevensmodel"
+date: '06-05-2020'
+weight: 10
+---
+
+Onderstaand het gegevensmodel van de API's voor Zaakgericht werken in XMI formaat. Hiermee is het onder andere leesbaar en bewerkbaar in Enterprise Architect en andere pakketen die het XMI formaat ondersteunen.
+
+Het gegevensmodel in XMI formaat: [Gegevensmodel API's voor Zaakgericht Werken dd. 1 mei 2020](_assets/20200501_ZGW_APIs.xml)
+
+De dtd voor het XMI formaat, deze plaatst u in dezelfde folder als het gegevensmodel: [UML_EA.DTD](UML_EA.DTD)
+
+Mocht u niet de beschikking hebben over het pakket Enterprise Architect biedt leverancier Sparx Systems een EA Viewer om het model in alleen-lezen modus te kunnen bekijken. Deze is [hier](https://www.sparxsystems.eu/enterprise-architect/ea-lite-edition/) te downloaden.
```

Comment:
Prima. Ik vermijdt meestal "klik hier", "zie hier" waar "hier" een link is.
In plaats daarvan maak ik een link van het woord/zinsdeel waar het om gaat, in dit geval "EA Viewer".

Maar dat stamt uit mij SEO-dagen.